dropwizard-jsonapi/src/main/java/io/rtr/jsonapi/bundle/ApiBundle.java

```diff
@@ -12,6 +13,7 @@
 	@Override
 	public void initialize(Bootstrap<?> bootstrap) {
 		bootstrap.getObjectMapper().addMixInAnnotations(Object.class, FieldFilterMixIn.class);
+		bootstrap.getObjectMapper().setSerializationInclusion(JsonInclude.Include.NON_EMPTY);
```

Comment:
Oh yeah, you are right, this was left over from some attempts at not showing attributes